src/client/containers/PullRequestSummary/ReviewSection/ReviewerList/index.js

```diff
@@ -43,9 +43,9 @@ class ReviewerList extends Component {
               />
             </OverlayTrigger>
             {this.props.reviews.map((r, i) => (
-              <span key={_.uniqueId()}>
+              <span key={_.uniqueId(r.user.username)}>
```

Comment:
Quick question, does this need to be globally unique? If not then `username` is sufficient?